doc/guide/api-cockpit.xml

```diff
@@ -1456,6 +1456,62 @@
 
   </refsection>
 
+<refsection id="latest-permission">
+    <title>Permission Lookup</title>
+
+    <para>Cockpit provides a mechnisim for checking if the current user satisfies a
+      given criteria. Currently capable of checking for root users, and group
+      membership. This is meant for updating UI elements based on what actions the
+      user can perform. It is <emphasis>not an access control mechanism</emphasis>.</para>
+
+    <refsection id="latest-permission-constructor">
+      <title>cockpit.permission()</title>
+<programlisting>
+ permission = cockpit.permission([options])
+</programlisting>
+
+    <para>Create a new permission object. If specified the <code>options</code> argument
+      object with a <code>"group"</code> group field. If no argument is given, the permission
+      will just check if the user is root.</para>
```

Comment:
This doesn't make it super clear that root is always given permission.  What about
```
The "root" user is always given permission.  The <code>options</code> argument can contain a <code>group</name> field, and members of that group are also given permission.
```